### PR TITLE
Patch release v1.1.1

### PR DESCRIPTION
**Pull request by ramprasadn**
## Description

- Configures `bcftools merge` to generate compressed vcf files.
- Fixes errors thrown by `bcftools concat` due to sample names in input vcf files not being in same order. 
    - GATK's MergeVcfs can handle such differences hence, the switch. However, GATK complains about spaces in the INFO columns so I have added a small groovy function to create a reformatted vcf file. 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.1 - Abu (Patch) [xxxx-xx-xx]
```
Comment (asp8200) [inline CHANGELOG.md:6]: Just a gentle reminder to add the date. (Perhaps you were planing to add that after approval of PR?)

File: workflows/raredisease.nf [hunk 2]
```diff
@@ -483,16 +483,9 @@ workflow RAREDISEASE {
...
-                .concat(ANALYSE_MT.out.tbi)
-                .groupTuple()
-                .set { ch_merged_tbi }
-
-            ch_merged_vcf.join(ch_merged_tbi, failOnMismatch:true, failOnDuplicate:true).set {ch_concat_in}
-
-            BCFTOOLS_CONCAT (ch_concat_in)
-            ch_snv_annotate = BCFTOOLS_CONCAT.out.vcf
-            ch_versions = ch_versions.mix(BCFTOOLS_CONCAT.out.versions)
+            GATK4_MERGEVCFS (ch_merged_vcf, [[],[]])
```
Comment (asp8200) [inline workflows/raredisease.nf:486]: Will the resulting VCF-file be sorted? If not, then should it perhaps be sorted?

File: CHANGELOG.md (hunk 1 above)
Comment (ramprasadn) [reply CHANGELOG.md:6]: Yupe, I usually do that after the approval :)

File: workflows/raredisease.nf (hunk 2 above)
Comment (ramprasadn) [reply workflows/raredisease.nf:486]: Good point, Anders! MergeVcfs does generate sorted vcfs. I noticed that it does so in my tests, and now on looking at the tool documentation more closely, it says so in the description as well, https://gatk.broadinstitute.org/hc/en-us/articles/360037226612-MergeVcfs-Picard- :)

**Review by asp8200 — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (jfy133) [inline CHANGELOG.md:6]: Set date!

**Review by jfy133 — APPROVED** [review]

**Comment by ramprasadn** [comment]
Thanks for the review @asp8200 and @jfy133 👍🏻

File: CHANGELOG.md (hunk 1 above)
Comment (ramprasadn) [reply CHANGELOG.md]: Done!

**Pull request merged**